include/RAJA/policy/atomic_builtin.hpp

```diff
@@ -38,6 +38,7 @@ namespace detail
 
 #if defined(RAJA_COMPILER_MSVC)
 
+RAJA_HOST_DEVICE
```

Comment:
Why have these been marked host device?

Reply:
Old comment I realize, but IIRC hip offers the builtins directly from clang, so it actually makes some good sense.